db/migrate/20160105161831_create_locations.rb

```diff
@@ -0,0 +1,22 @@
+class CreateLocations < ActiveRecord::Migration
+  def change
+    create_table :locations do |t|
+      t.float :latitude, null: false
+      t.float :longitude, null: false
+      t.float :height
+      t.timestamp :registration_time, null: true, index: true
+      t.belongs_to :sensor, null: false, index: true, foreign_key: true
+
+      t.timestamps null: false
+    end
+
+    Location.reset_column_information
+
+    Sensor.find_each do |sensor|
```

Comment:
Similar like in in `AddTokenToSensor` migration - this is not reversible thus `up` method.